mediator/src/middlewares/schemas/service-request.ts

```diff
@@ -1,5 +1,6 @@
 import joi from 'joi';
 
 export const createServiceSchema = joi.object({
-  patient_id: joi.number().required()
+  patient_id: joi.number().required(),
```

Comment:
@njogz, when posting the patient/callbackURL payload to service-request, as advised in [your step-by-step guide](https://github.com/medic/interoperability/issues/26#issuecomment-1441451470),  I get:

>     "message": "\"patient_id\" must be a number",

and that is normal, as the schema required a number and not a uuid. anything that I am missing here?